sortinghat/core/schema.py

```diff
@@ -133,6 +136,23 @@ class Meta:
         model = Enrollment
 
 
+class AffiliationResultType(graphene.ObjectType):
+    uuid = graphene.String()
+    organizations = graphene.List(graphene.String)
```

Comment:
I think `organizations` should be defined as a list of `OrganizationType`. Is there a specific reason for setting it as a list of strings?

Reply:
The results generated by the job return the name of the organizations an individual was affiliated. As the results are stored in a Redis queue, strings work better that objects because with complex objects we would have to define methods to marshall and unmarshall when they are read/written.

What we can do is to convert the name to an `OrganiationType` in the graphene resolver but I not sure how to do it now. Let's keep this as is now and let's think about it to improve this feature.